api/swagger.json

```diff
@@ -4834,7 +4834,13 @@
   },
   "components": {
     "schemas": {
-      "FaceDetectorConfig": {
+      "VideoFaceDetectorConfig": {
+        "required": [
```

Comment:
Lot of required field definitions has been added or removed...
Why is it so ?

Reply:
TL;DR bugfix

Face (and Pedestrian) detector config models used in updating Cameras (videos) were split from those used in other API endpoints (watchlist search/register).

This was done so that all properties of the config are required for camera updates, whereas the properties for the other endpoints are all optional (using default when not specified).

Before this changes you could update Camera with only partially filled face detector config, and this caused the not-specified config properties to revert to default value.